script.js

```diff
@@ -0,0 +1,134 @@
+let storedNotes = JSON.parse(localStorage.getItem("notes"));
+const list = document.getElementById("list");
+let notes = storedNotes?storedNotes:[];
+showNotes(); 
+
+document.getElementById("add-btn").addEventListener("click",function(){
```

Comment:
if the user tried to add input with just spaces on it, and no characters, it will add an empty list item. for example, try to press `space` more than one time on the input field and see what will happen.
It would be great if you handle this case!

Reply:
There is also another case:
 - suppose you added an input like this with spaces at the beginning.
![image](https://user-images.githubusercontent.com/54964739/97785824-2abaa080-1bb0-11eb-9dec-8b8d2cfd1b40.png)

- After that add another input with the letter `a` only.
![image](https://user-images.githubusercontent.com/54964739/97785897-779e7700-1bb0-11eb-8c48-5b57d71acd24.png)

- Now click `sort`, the output will be like this:
![image](https://user-images.githubusercontent.com/54964739/97785906-89801a00-1bb0-11eb-8631-1a1333c34583.png)

Try to fix this one too.

Reply:
I only have these two comments, otherwise, everything looks really GREAT!! the code is very clean and easy to understand.

Reply:
Right away 👍🤞

Reply:
Thank you 👌